app/pods/cluster/template.hbs

```diff
@@ -37,39 +37,24 @@
   {{/dashboard-module}}
 
   {{#dashboard-module label='Nodes'}}
-    {{#if model.nodes.length}}
-      <table class="table">
-        <thead>
-        <tr>
-          <th>Ping</th>
-          <th>Stats</th>
-          <th>Node</th>
-        </tr>
-        </thead>
-        <tbody>
-        {{#each model.nodes as |node|}}
-          <tr>
-            <td>
-              {{#link-to 'riak-ping' (query-params nodeId=node.id clusterId=model.clusterId)
-              classNames='btn btn-sm btn-primary' }}
-                Node Ping
-              {{/link-to}}
-            </td>
-            <td>
-              {{#link-to 'node-stats' (query-params nodeId=node.id clusterId=model.clusterId)
-              classNames='btn btn-sm btn-primary' }}
-                Node Stats
-              {{/link-to}}
-            </td>
-            <td>
-              {{node.id}}
-            </td>
-          </tr>
-        {{/each}}
-        </tbody>
-      </table>
+    {{#if model.riakNodes}}
+      <ul class="button-list">
+          {{#each model.riakNodes as |node|}}
+              <li>
+                  {{#link-to 'node-stats'
+                  (query-params nodeId=node.id clusterId=model.id)
+                  classNameBindings="!node.available:danger"}}
+                      {{node.id}}
+                      <span {{bind-attr class=":glyphicon node.available:glyphicon-ok-circle:glyphicon-ban-circle"}}
```

Comment:
No, I have JShint running, and should be catching all of these. In terms of deprecations & warnings, they don't output in my console in red the way a proper error does, so I must have missed it. I'll be better about this, thanks for the careful eye.